tests/helper/helper_telemetry.go

```diff
@@ -0,0 +1,29 @@
+package helper
+
+import (
+	"encoding/json"
+	_ "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/redhat-developer/odo/pkg/segment"
+	"io/ioutil"
+)
+
+func CreateTelemetryDebugFile() {
```

Comment:
Can you add doc strings for the functions defined in this file?

Reply:
done